store/src/components/productItem/productItem.ts

```diff
@@ -0,0 +1,35 @@
+import { getCarts } from '../../services/api/api';
+import { Trash } from '../../types';
+
+export async function getCount() {
+    let res;
+    const count = document.querySelector('.header__div--hidden') as HTMLElement;
+    await setStorage();
+    if (String(sessionStorage.getItem('trash'))) {
+        await setStorage();
+        res = JSON.parse(String(sessionStorage.getItem('trash')));
+    }
+    if (res) {
+        count.classList.add('cart-count');
+        res.forEach((i: Trash) => getProductBasket(i.productId));
+        count.textContent = JSON.parse(String(sessionStorage.getItem('trash'))).length;
+    }
+}
+async function setStorage() {
+    await getCarts().then(({ body }) =>
+        body.results[0]?.lineItems
+            ? sessionStorage.setItem('trash', JSON.stringify(body.results[0]?.lineItems))
+            : sessionStorage.setItem('trash', JSON.stringify(null))
+    );
+}
+function getProductBasket(param: string) {
+    const arr = document.getElementById(param);
+    if (arr?.children[4]) {
+        addButton(arr?.children[4]);
+    }
+}
```

Comment:
doesn't looks obviously what is it doing. few questions:
- this function isn't doing and doesn't match to its name. Based on naming(getProductBasket), it should get some productBasket, can accept some order/param and that's it. But it's doing side effect
- has some conditions where it contains some magic value. 
- why this getProductBasket function is triggering `addButton`? This also makes a little confusion 

Every function should do only one thing or at least should do as described on its naming
Please have a look to Single Responsibility principle